src/map/entities/charentity.h

```diff
@@ -34,6 +34,7 @@ This file is part of DarkStar-server source code.
 
 #include "battleentity.h"
 #include "petentity.h"
+#include "trustentity.h"
```

Comment:
It can't without compile errors.

Reply:
hmm, I think vector must need to know the destructor, then.  If vector needs it, then I suppose we leave it

Reply:
strange... looks like it's only being used for pointers and nothing else so `class CTrustEntity;` should work unless visual studio is playing up

you'll still need to include `trustentity.h` in `charentity.h` though, since all `class CTrustEntity;` tells the compiler is that it's a type, but doesnt provide it with its implementation details

Reply:
you'll need to include `trustentity.h` in `charentity.cpp` even, edited typo in previous commit (for you email readers)

Reply:
Done in https://github.com/DarkstarProject/darkstar/pull/4729/commits/948344cef06c3569742b020b432e96d18d8280dc